app/controllers/users/names_controller.rb

```diff
@@ -29,4 +30,8 @@ def authorize_user
   def user_params
     params.require(:user).permit(*current_user.permitted_params.intersection([:name]))
   end
+
+  def redirect_to_account_page_if_acting_on_own_user
+    redirect_to edit_account_email_path if current_user == @user
```

Comment:
I don't think it makes sense to redirect to the account edit email page here. And we don't have an account edit name page so maybe it should just redirect to the account path?

Reply:
I've addressed this in the commit titled, "Redirect to account path if user tries to edit their own name".

Reply:
Can you explain why you don't think it makes sense to redirect to the account edit email page here? Is it purely because we can't be consistent with how the same thing works for editing a user's name? Or is there some other reason. I looked at the commit you added, but that commit note doesn't explain this either.

Reply:
> Can you explain why you don't think it makes sense to redirect to the account edit email page here?

It's because it feels odd to me that if you're notionally* trying to edit your own name via the users edit page that we'd redirect you to a page where the only thing you can do is edit your email address. Does that make sense?

Reply:
Doh! Sorry - I saw the title of the PR was about editing the user's email and assumed this comment related to the `Users::EmailsController`. Please ignore me - and thanks for catching it! 😊